core/platform/models.py

```diff
@@ -27,8 +27,16 @@
 
 
 class _Platform(object):
+    """A base class for platform-specific imports related to GAE.
+    """
     @classmethod
     def import_models(cls):
+        """An abstract method that should be implemented on inherited
+        classes.
+
+        Raises:
+            NotImplementedError: The method not overwritten in derived classes.
```

Comment:
@joydeep1701 Can you make it as (add an "is" after method)
"NotImplementedError: The method is not ...."